CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
```

Comment:
Do we even need to maintain CHANGELOG.md?
GitHub Releases have release notes

Reply:
Yes, but I'd like the information to be accessible locally. Maybe it's not worth it, I dunno.

Reply:
Yeah, I was considering the same. While a change-log _can_ be nice, it also easily gets forgotten to be updated, or adds additional maintenance.

For a project that produces binaries (such as runc), I could see it being more relevant (also for potential inclusion in `deb`, `rpm` packages), but a bit on the fence for a library module like `cgroups`

Reply:
This kind of make sense. We can maintain the changelog under https://github.com/opencontainers/cgroups/releases